### PR TITLE
Added functionality to keep the number of track divisions constant

### PR DESCRIPTION
**Pull request by cwjames1983**

### CODE REVIEW

File: SmallScale/FacetPropagation/rough.in [hunk 1]
```diff
@@ -1,17 +1,17 @@
-3 0.1 1e-8 1 1 1e-6 0 0 1000 
-1 ../Surfaces/trivial_128.dat
+3 0.1 1e-8 1 1 1e-6 0 0 0
```
Comment (cwjames1983) [inline SmallScale/FacetPropagation/rough.in:1]: Can you please revert this to trivial_128 (and example_track) below?

File: SmallScale/FacetPropagation/rough.in [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+1 Surfaces/trivial_1024.dat
 1 0 0 0 
 0 modded_trivial.dat
-../TrackFiles/example_track.dat
+TrackFiles/trivial_tracks.dat
 0   0    0.0    0.0    50.0
 1.8 1.8 18e10
-0 0 5
```
Comment (cwjames1983) [inline SmallScale/FacetPropagation/rough.in]: Can you keep the default to be the simple phi=0 case?

**Review by cwjames1983 — COMMENTED** [review]
Please just modify the infile then we'll merge!

**Review by lisa-holland — APPROVED** [review]
Have made the requested changes to the input files.

**Pull request merged**